cfme/tests/intelligence/reports/test_metering_report.py

```diff
@@ -0,0 +1,314 @@
+# -*- coding: utf-8 -*-
+import fauxfactory
+import math
+import pytest
+import re
+
+from datetime import date
+
+from cfme import test_requirements
+from cfme.base.credential import Credential
+from cfme.common.vm import VM
+from cfme.common.provider import BaseProvider
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.intelligence.reports.reports import CustomReport
+from cfme.utils.log import logger
+from cfme.utils.wait import wait_for
+
+
+pytestmark = [
+    pytest.mark.tier(2),
+    pytest.mark.ignore_stream('5.8'),
+    pytest.mark.provider([VMwareProvider, RHEVMProvider],
+                         scope='module',
+                         required_fields=[(['cap_and_util', 'test_chargeback'], True)]),
+    test_requirements.chargeback,
+]
+
+
+@pytest.yield_fixture(scope="module")
+def clean_setup_provider(request, has_no_providers_modscope, setup_provider_modscope,
+        provider):
+    yield
+    BaseProvider.clear_providers()
+
+
+def new_credential():
```

Comment:
**Optional**
I don't see the need for this being a function, since its a single class instantiation with a call to fauxfactory. It isn't doing anything more (logging, version picking, credential lookup) than could be done in a statement as a local variable or directly in-line where its used now.

Reply:
DOne